src/styles/blocks/header-block/header.scss

```diff
@@ -0,0 +1,14 @@
+@import "./nav-block.scss";
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  height: $heigth-container;
+  padding-right: $padding-for-container;
+  padding-left: $padding-for-container;
+}
+
+.photo {
```

Comment:
if it's a separate bem block you should create a new file for it

however, it will be better to make it some bem element